packages/twenty-server/src/modules/workflow/workflow-status/utils/workflow-status-combination.util.ts

```diff
@@ -0,0 +1,135 @@
+import { WorkflowVersionStatus } from 'src/modules/workflow/common/standard-objects/workflow-version.workspace-entity';
+import { WorkflowStatus } from 'src/modules/workflow/common/standard-objects/workflow.workspace-entity';
+
+export enum WorkflowStatusCombination {
+  ACTIVE = 'ACTIVE',
+  DRAFT = 'DRAFT',
+  DEACTIVATED = 'DEACTIVATED',
+  ACTIVE_AND_DRAFT = 'ACTIVE_AND_DRAFT',
+  DEACTIVATED_AND_DRAFT = 'DEACTIVATED_AND_DRAFT',
+  NO_STATUSES = 'NO_STATUSES',
+}
+
+export const getWorkflowStatusCombinationFromArray = (
+  statuses: WorkflowStatus[],
+): WorkflowStatusCombination => {
+  if (
+    statuses.includes(WorkflowStatus.ACTIVE) &&
+    statuses.includes(WorkflowStatus.DRAFT)
+  ) {
+    return WorkflowStatusCombination.ACTIVE_AND_DRAFT;
+  }
```

Comment:
I would be more assertive here:
```
if (
statuses === [WorkflowStatus.ACTIVE, WorkflowStatus.DRAFT]
  ) {
    return WorkflowStatusCombination.ACTIVE_AND_DRAFT;
  }
...
```
Would be even nicer to create constants to store combinations:
```
const ACTIVE_AND_DRAFT_COMBINATION = [WorkflowStatus.ACTIVE, WorkflowStatus.DRAFT]
...
```
in order to use them in `getWorkflowStatusesFromCombination`